articles/idam/idam.md

```diff
@@ -0,0 +1,83 @@
+---
+title: ID & Access Management API | UKCloud Ltd
+description: Provides information on managing permissions using the ID & Access Management API
+services: idam
+author: Simon Fish
+
+toc_rootlink: Getting Started
+toc_sub1:
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: IDAM
+toc_fullpath: Getting Started/idam.md
+toc_mdlink: idam.md
+---
+
+# Identity and Access Management (IDAM)
+
+UKCloud's **ID** & **A**ccess **M**anagement API (IDAM) allows users to manage user access to products within their account. Adding a user to a group changes which permissions they have.
```

Comment:
No need to use bold to highlight the letters of the acronym
Change "ID" to "Identity"
Change "allows users" to "enables you" (allows implies permission rather than ability)
Change "their" to "your"